CMakeLists.txt

```diff
@@ -1,14 +1,15 @@
-cmake_minimum_required(VERSION 3.0.0)
+﻿cmake_minimum_required(VERSION 3.0.0)
 
 # edit this to wherever your vcpkg is
-set(VCPKG_DIR ../vcpkg CACHE PATH "path to vcpkg root")
```

Comment:
keep it ../vcpkg, user can change while configuring cmake

Reply:
well this can be kept anything, since user can change :P